docs/sources/operators-guide/deploy-grafana-mimir/jsonnet/deploying.md

```diff
@@ -9,6 +9,8 @@ weight: 10
 
 # Deploying Grafana Mimir with Jsonnet and Tanka
 
+> **Note**: Jsonnet-based installation assumes that you have support for `EtcdCluster` resource already installed in your kubernetes.
```

Comment:
I am not sure does this make sense because maybe we need to add some tips for how to install it.  I'm also not sure what we should add to help users add it.

Reply:
I am also curious why don't we add it in our mimir jsonlib?

Reply:
Good point. Jsonnet uses etcd for the HA tracker by default. HA tracker is optional (I mean you will not have any impact unless you use the HA tracker). To install etcd, we expect you have the etcd-operator https://github.com/coreos/etcd-operator running in your K8S cluster.

I would suggest you move the  `> **Note**` at the end of this section, and clarify it based on what I've mentioned ☝️

Reply:
Thanks! I merged two notes. How about now?